core/src/commonMain/resources/MR/base/strings.xml

```diff
@@ -160,6 +160,18 @@
     <string name="onboarding_headline3">Stay organized</string>
     <string name="onboarding_subheadline3">Education content, assignments, classes, and attendance all in one place</string>
     <string name="download_continue_stacked_label">Continue download</string>
+
+    <string name="onboarding_individual_text">I\'m an individual learner</string>
```

Comment:
Please make the string match at least the first few words e.g. name="im_an_individual_learner". 

These are basically new strings. We don't use the prefixes anymore (because that makes it harder to find strings to reuse).